src/dateparser/dateparser.js

```diff
@@ -81,7 +105,7 @@ angular.module('ui.bootstrap.dateparser', [])
       return input;
     }
 
-    format = $locale.DATETIME_FORMATS[format] || format;
+    format = $locale.DATETIME_FORMATS[format] || format.replace(/:/g, '\\:');
```

Comment:
I think tests will fail if a DATETIME_FORMAT from $locale containing `:` is used.
Should probably add some of those formats in the dateparser.spec below (e.g. a format of `short`).